src/Var.jl

```diff
@@ -966,6 +971,203 @@ function split_by_season(var::OutputVar)
+        ret_attributes["long_name"] *= " " * short_name(sim)
+    end
+
+    # Compute global bias and store it as an attribute
+    integrated_bias = integrate_lonlat(bias).data
+    normalization =
+        integrate_lonlat(
+            OutputVar(
+                bias.attributes,
+                bias.dims,
+                bias.dim_attributes,
+                ones(size(bias.data)),
+            ),
+        ).data
+    # Do ./ instead of / because we are dividing between zero dimensional arrays
+    global_bias = integrated_bias ./ normalization
+    ret_attributes["global_bias"] = global_bias
+    return OutputVar(ret_attributes, bias.dims, bias.dim_attributes, bias.data)
+end
+
+"""
+    global_bias(sim::OutputVar, obs::OutputVar)
+
+Return the global bias of `data` in `sim` and `obs` over longitude and latitude.
+
+This function is currently only implemented for `OutputVar`s with only the dimensions
+longitude and latitude. Units must be supplied for data and dimensions in `sim` and `obs`.
+The units for longitude and latitude should be degrees. Resampling is done automatically by
+resampling `obs` on `sim`.
+"""
+function global_bias(sim::OutputVar, obs::OutputVar)
+    bias_var = bias(sim, obs)
+    return bias_var.attributes["global_bias"]
+end
+
+"""
+    squared_error(sim::OutputVar, obs::OutputVar)
+
+Return a `OutputVar` whose data is the squared error (`(sim.data - obs.data)^2`) and compute
+the global mean squared error and global root mean squared error of `data` in `sim` and
+`obs` over longitude and latitude which can found in `var.attributes["mse"]` and
+`var.attributes["rmse"]`.
+
+This function is currently implemented for `OutputVar`s with only the dimensions longitude
+and latitude. Units must be supplied for data and dimensions in `sim` and `obs`. The units
+for longitude and latitude should be degrees. Resampling is done automatically by resampling
+`obs` on `sim`. Attributes in `sim` and `obs` will be thrown away. The long name and short
+name of the returned `OutputVar` will be updated to reflect that a squared error is computed.
+"""
+function squared_error(sim::OutputVar, obs::OutputVar)
+    _check_sim_obs_units_consistent(sim, obs)
+
+    # Resample obs on sim to ensure the size of data in sim and obs are the same and the
+    # dims are the same
+    obs_resampled = resampled_as(obs, sim)
+
+    # Compute squared error
+    # Do not use ^ since ^ is not defined between a OutputVar and Real
+    squared_error = (sim - obs_resampled) * (sim - obs_resampled)
+
+    # Do this because we do not want to store global mse and rmse as strings
+    ret_attributes = Dict{String, Any}(squared_error.attributes)
+
+    # Add units back for bias
+    # Always treat as a string since the string representation of something type Unitful is
+    # not always parseable as a Unitful.Unit (see:
+    # https://github.com/PainterQubits/Unitful.jl/issues/412)
+    ret_attributes["units"] = "(" * units(sim) * ")^2"
```

Comment:
You can also write this as
```suggestion
    ret_attributes["units"] = "($(units(sim)))^2"
```